drivers/hmis/app/graphql/types/hmis_schema/assessment.rb

```diff
@@ -113,6 +113,7 @@ def disability_group
 
       # Build OpenStruct for DisabilityGroup type
       OpenStruct.new(
+        id: object.id, # for logging
```

Comment:
This will store the Assessment ID, like `DisabilityGroup/1`. I guess that's OK -- it doesn't tell us which Disability records were resolved, but we could look at what's linked to the Assessment to try to figure it out if needed.